frontend/src/column.tsx

```diff
@@ -15,15 +9,17 @@ interface ColumnProps {
 
 export function Column(props: ColumnProps) {
    return (
-      <Card>
-         <Card.Header>{props.title}</Card.Header>
-         <ColumnBody>
-            <ListGroup>
-               {props.pulls.map((pull) =>
-                  <PullCard key={pull.getKey()} pull={pull}/>
-               )}
-            </ListGroup>
-         </ColumnBody>
-      </Card>
+      <Box border="1px" borderRadius={7} overflow="hidden">
+         <Flex bgColor="var(--panel-default-background)" size="m">
+            <Box p={4}>{props.title}</Box>
+            <Spacer/>
+            <Box p={4} borderLeft="solid 1px rgba(0,0,0,0.3)" >{props.pulls.length}</Box>
+         </Flex>
```

Comment:
This is kind of the pattern I've followed when working with Chakra, where I mostly use chakra components and I define styles by passing literals to props.

Reply:
So freaking ugly though. Gonna end up with our markup being 90% styling, thus making it really hard to read the structure.

Reply:
Yea it does obfuscate the structure. You might consider looking at ink of pixel's work for examples too, maybe they've come up with some good patterns

Here's a particularly verbose example 😆  https://github.com/iFixit/react-commerce/blob/main/frontend/features/productList/components/sections/FeaturedProductListSection.tsx

Reply:
It seems hard to balance "where is this style coming from" with "what does the structure look like" with a lot of these theme libraries/css in general